pkg/porter/archive.go

```diff
@@ -22,12 +23,12 @@ type ArchiveOptions struct {
 }
 
 // Validate performs validation on the publish options
-func (o *ArchiveOptions) Validate(args []string, ctx *context.Context) error {
+func (o *ArchiveOptions) Validate(args []string, runtime cnabprovider.CNABProvider, ctx *context.Context) error {
 	if len(args) < 1 || args[0] == "" {
 		return errors.New("Destination File is required")
 	}
 	o.ArchiveFile = args[0]
-	return o.BundleLifecycleOpts.Validate(args, ctx)
+	return o.BundleLifecycleOpts.Validate(args, runtime, ctx)
```

Comment:
Yeah I think seeing passing the runtime to a client side function (Validate) makes it clear that LoadParameterSet really belongs on Porter, or somewhere else in a client package instead of on the runtime.